web/src/components/Wrapper.js

```diff
@@ -17,7 +20,13 @@ class Wrapper extends Component {
 
   render() {
     const { children } = this.props;
-    return <div className="site">{children}</div>;
+    let showSiteTitle = location.pathname === '/'; // eslint-disable-line prefer-const
```

Comment:
Can this go ahead and be a `const`?

Reply:
I ran into some issues with it being a `const` because the wrapper doesn't always rerender itself... or something? It was failing to reassign the value when I navigated between pages, so it was just always equal to `/`. If there's a different way to approach it, I'm all ears!

Reply:
Ahhhhhh.  I don't know why using a `let` would fix that, but the problem makes sense.  If none of the wrapper's props change, it won't re-render.  If you use `withRouter()`, the component should re-render whenever the location changes because the location props are passed in.